src/IB/ConstraintIBMethod.cpp

```diff
@@ -452,6 +454,7 @@ ConstraintIBMethod::postprocessSolveFluidEquations(double current_time, double n
     if (d_output_torque) calculateTorque();
     if (d_output_eul_mom) calculateEulerianMomentum();
     if (d_output_power) calculatePower();
+    calculateLinearMomentumStructure();
```

Comment:
No, since the linear momentum is not outputted to a file. Rather, it is computed then stored in d_linear_mom_structure, which is returned by getLinearMomentumStructure(). This is helpful for hydrodynamic force computation since the momentum of the structure is needed at every time step.